shared/tests/utility/math/docs/genValsInRange.m

```diff
@@ -0,0 +1,6 @@
+function randVals = genValsInRange(min, max, total, cols)
+%This function generates random values in the given range
+%params: min = lower val, max = upper val, total = total numbers generated
```

Comment:
I would suggest putting the params on separate lines in this comment. On one line, it's a little harder for me to parse than I think it could be